src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { snazzyButtonStyle } from './helpers';
+
+
+interface ButtonProps {
+  children: React.ReactNode;
+}
+
+const styles = {
+  snazzyButton: {
+
+  }
+};
+
+const useStyles = createUseStyles(styles);
+
+interface SnazzyButtonProps {
+  isSnazzy: boolean;
+}
```

Comment:
Without adding children to this type, you will get a type-error and be unable to proceed.

```suggestion
interface SnazzyButtonProps {
  isSnazzy: boolean;
  children: React.ReactNode;
}
```